app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,7 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if can? :access, Admin
```

Comment:
Actually, this line ```https://github.com/raluka/osem/blob/view_comment_notification/app/views/layouts/_user_menu.html.haml#L27``` is about **index**, not **show**, and it is following the logic from Admin::ConferenceController  #index. But I will add an extra condition for "Create New" link, for the future.